jupyterbook/content/ioos_installation_conda.md

```diff
@@ -172,7 +172,7 @@ Check out [Travis Oliphant's blog piece](http://technicaldiscovery.blogspot.com/
 
 ### What to do when everything seems to be broken?
 
-For example, ff you are seeing kernel erros like the one below.
+For example, if you are seeing kernel erros like the one below.
```

Comment:
erros -> errors

Reply:
Jeez, I cannot even copy-n-paste this afternoon! Sorry.